sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -467,8 +467,8 @@ abstract class StringPredicate extends BinaryExpression
  */
 @ExpressionDescription(
   usage = """
-    _FUNC_(expr1, expr2) - Returns a boolean value if expr2 is found inside expr1.
-    Returns NULL if either input expression is NULL.
+    _FUNC_(expr1, expr2) - Returns a BOOLEAN. The value is True if right is found inside left.
```

Comment:
I think using the words expr1 and expr2 is fine as that's shown in the documentation together. BOOLEAN -> boolean is unnecessary. Let's change `Otherwise, returns False.` alone